### PR TITLE
Add manual testing

### PR DESCRIPTION
**Pull request by zognin**
Part of #206 

Can be viewed [here](https://zognin.github.io/tp/DeveloperGuide.html#appendix-instructions-for-manual-testing)

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1039,44 +1039,74 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `delete-ri 1 i/abc`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri 1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri i/1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+1. Editing a recipe ingredient
+    1. Prerequisites: Recipe bookmarks list shows at least 1 recipe and at most 3 recipes.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1062]: Need to include that there's no existing duplicate recipe? And test case failure when trying to edit a recipe when it already exists
```suggestion
    1. Prerequisites: Recipe bookmarks list only has these 3 recipes:
        * name: Pecan pie, price: $10
        * name: Banana split, price: $10
        * name: Orange cake, price: $12
```

**Review by EuniceSim142 — COMMENTED** [review]
LGTM, just some minor nits.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1039,44 +1039,74 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `edit-r 1 rn/Truffle fries rp/8.00`
+       <br>Expected: First recipe in the recipe bookmarks list is edited to have recipe name 'Truffle fries' and recipe price of '$8.00'.
+       Its position in the recipe bookmarks list may change. Details of the edited recipe shown in the result display box.
```
Comment (Imranr2) [inline docs/DeveloperGuide.md:1065]:
```suggestion
       Its position in the recipe bookmarks list may change. Details of the edited recipe is shown in the result display box.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1039,44 +1039,74 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `edit-r 1 rn/Apple pie`
+       <br>Expected: First recipe in the recipe bookmarks list is edited to have recipe name 'Apple pie'.
+       Its position in the recipe bookmarks list may change. Details of the edited recipe shown in the result display box.
```
Comment (Imranr2) [inline docs/DeveloperGuide.md:1068]:
```suggestion
       Its position in the recipe bookmarks list may change. Details of the edited recipe is shown in the result display box.
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1039,44 +1039,74 @@ testers are expected to do more *exploratory* testing.
...
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `edit-r abc`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `edit-r 1 rp/-2`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+1. Finding recipe by keywords
+    1. Prerequisites: There are exactly 2 recipes in the recipe bookmarks list.
+       One has the recipe name "Apple Pie", the other has the recipe name "Banana split".
+    1. Test case: `find-r rn/apple`
+       <br>Expected: Recipe bookmarks list only shows the recipe with the recipe name "Apple Pie".
```
Comment (Imranr2) [inline docs/DeveloperGuide.md:1083]:
```suggestion
       <br>Expected: Recipe bookmarks list only shows the recipe with the recipe name 'Apple Pie'.
```

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zognin) [reply docs/DeveloperGuide.md:1062]: Oh that's true. I think it's better to not include those cases because it becomes additional work to explain.

Comment (EuniceSim142) [reply docs/DeveloperGuide.md]: Ok that's fine. Anyways this is just a guide, it doesn't need to be exhaustive.

**Review by Imranr2 — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1201,44 +1201,77 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `delete-ri i/1`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+    1. Test case: `delete-ri`
+       <br>Expected: No recipe is edited. Error details shown in the result display box.
+1. Editing a recipe
+    1. Prerequisites: Recipe bookmarks list only has these 3 recipes:
+        * name: Banana split, price: $10
+        * name: Orange cake, price: $12
+        * name: Pecan pie, price: $10
+     1. Test case: `edit-r 1 rn/Truffle fries rp/8.00`
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1228]: The '1.' is one space behind

**Review by EuniceSim142 — COMMENTED** [review]
LGTM, just one small change.

**Review by EuniceSim142 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**